crates/database/src/settlement_observations.rs

```diff
@@ -90,5 +92,26 @@ mod tests {
         .unwrap()
         .unwrap();
         assert_eq!(input, output);
+
+        let new_input = Observation {
+            gas_used: 5.into(),
+            effective_gas_price: 6.into(),
+            surplus: 7.into(),
+            fee: 8.into(),
+            block_number: 1,
+            log_index: 1,
```

Comment:
Nit: I'd like to make it clear that these values come from `input`: maybe `block_number: input.block_number(), log_index: input.log_index()`. If they were different, the test would pass anyway and we might not notice a change.